app/src/main/java/com/restaurantroulette/ui/main/MainViewModel.kt

```diff
@@ -3,6 +3,7 @@ package com.restaurantroulette.ui.main
 import androidx.lifecycle.ViewModel
 
 class MainViewModel : ViewModel() {
-    var randomBool: Boolean = false
-    // TODO: Implement the ViewModel
+    internal fun getRandomBool(): Boolean {
+        return Math.random() >= 0.5
```

Comment:
Ok... looks like this is returning true or false randomly, with about a 50% chance of either.